doc/kubevela-definitions/cardano/cardano-wallet-connector.md

```diff
@@ -0,0 +1,39 @@
+# Cardano Wallet Connector trait
+
+## Overview
+
+The Cardano Wallet Connector trait acts as an intermediary tool allowing seamless communication between your application and the Cardano wallet.
+
+The ***required*** parameter for implementing this trait is:
+ - `network`: Defines the network choice for the Cardano Wallet. The available options are "preview", "preprod" and "mainnet".
+
+The *Recreate* deployment strategy has been defined as the default strategy for this trait.
```

Comment:
```suggestion
Because the `cardano-wallet` stores its state in a persistent volume, this trait sets [Recreate](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#recreate-deployment) as the default deployment strategy.
```

Reply:
Because the `cardano-wallet` stores its state in a persistent volume, this trait sets [Recreate](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#recreate-deployment) as its deployment strategy.

Reply:
i'll put it as resolved but feel free to update it as you with.